docs/rfcs/0003_template_fields_mvp.md

```diff
@@ -0,0 +1,406 @@
+# 🗃 Template Field Definitions
+
+**Status**: Approved, waiting for implementation
+
+This document entails the specifications for template field definitions. Specifically:
+
+-   Components that define a field
+-   Encoding field definitions
+-   Template generation from field definitions
+-   Field validation
+
+## Background
+
+The phrase **"data submission template"** describes two things:
+
+### Template Submission File
+
+A tabular file that conforms to a standard format and
+has an associated set of submission rules for populating it.
+
+![Submission File](../_static/images/submission-template.png)
+
+### Template Field Definitions
+
+The submission rules, encoded as a set of fields where each field describes
+things like: column name, column type, values accepted,
+instructions for populating values, etc.
+
+![Field Definitions](../_static/images/template-field-defs.png)
+
+## High Level User Flow
+
+TODO: Add diagram?
+
+### 1. Build Template
+
+-   Admin: Build template on Data Tracker (e.g. define the template fields)
```

Comment:
How is a template built on Data Tracker? Is there an interface for that?

Reply:
Yea and its TBD right now. I may try an mock up something this week for it

Reply:
For the MVP, building the template may just mean uploading the field definitions in a TSV file or authoring the field definitions JSON in some sort of input field

Reply:
Cool I wasn't sure if I was missing something in this doc